common/config.js

```diff
@@ -131,6 +133,12 @@ function configDefaultsForEnv({marthaEnv, dsdeEnv = dsdeEnvFrom(marthaEnv)}) {
                         return HOST_MOCK_DRS;
                     case ENV_PROD:
                         return HOST_TERRA_DATA_REPO_PROD;
+                    case ENV_STAGING:
+                    case ENV_ALPHA:
```

Comment:
The TDR deployment must be in the same tier as Sam (and Martha, etc.) to work. This is because passing the auth info (i.e., token) from Sam in one tier to TDR (or any other Terra service) in another tier will fail.

The `terraDataRepoHost` block must map each supported tier to the TDR instance in that same tier, as shown below:
```
        terraDataRepoHost:
            (() => {
                switch (marthaEnv) {
                    case ENV_MOCK:
                        return HOST_MOCK_DRS;
                    case ENV_PROD:
                        return HOST_TERRA_DATA_REPO_PROD;
                    case ENV_STAGING:
                        return HOST_TERRA_DATA_REPO_STAGING;
                    case ENV_ALPHA:
                        return HOST_TERRA_DATA_REPO_ALPHA;
                    case ENV_DEV:
                    case ENV_CROMWELL_DEV:
                        return HOST_TDR_DEV;
                    default:
                        console.log(`Unsupported environment: '${marthaEnv}'.`);
                }
            })(),
  ```
I meant to put the above in the Slack message [here](https://broadinstitute.slack.com/archives/C01KXE68J5D/p1680181716980209?thread_ts=1679508094.938999&cid=C01KXE68J5D), yet I see I missed an update there and I understand the confusion. Hopefully this is clear now. 

Because all known/supported tiers are explicitly mapped to TDR in that same tier, I think the only reasonable `default` action is to issue a message indicating something is amiss.
I see `console.log` used for this elsewhere in the Martha code, so hopefully, that is appropriate here, too(?)

Reply:
Ah yeah I see I didn't quite get the nuance of that from your earlier comment, but this makes sense! I'll wait for you to finish reviewing then I'll update with changes. Thank you!